.github/workflows/dh-cd.yml

```diff
@@ -12,20 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: DH Backend CD
+# DataHub UI Frontend and BFF
+name: DH CD
 
 on:
   push:
     branches:
       - main
+  pull_request: {}
```

Comment:
This has some wierd connection to the is-affected that causes it to fail if not present.
I think its because somewhere they are checking if a variable exists on `pull_request.event` or something.

I wrote a comment that it can be removed once the other is removed.